### PR TITLE
[NUI] DragAndDrop: Add NotSupportedException for Multi-Window Feature

### PR DESCRIPTION
**Pull request by taehyub**
### API Changes ###

 - ACR:

### CODE REVIEW

**Review by jaehyun0cho — CHANGES_REQUESTED** [review]

File: src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs [hunk 1]
```diff
@@ -42,9 +45,28 @@ public class DragAndDrop : BaseHandle
         private int shadowWidth = 100;
         private int shadowHeight = 100;
 
+        static internal bool IsSupportedMultiWindow()
```
[the remark this answers is not in the document]
Comment (taehyub) [reply src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs]: done

File: src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs [hunk 2]
```diff
@@ -42,9 +45,28 @@ public class DragAndDrop : BaseHandle
...
+            }
+            return isSupported;
+        }
+
         [EditorBrowsable(EditorBrowsableState.Never)]
         private DragAndDrop() : this(Interop.DragAndDrop.New(), true)
         {
+            if (IsSupportedMultiWindow() == false)
+            {
+                throw new NotSupportedException("This device does not support surfaceless_context. So Window cannot be created.");
```
[the remark this answers is not in the document]
Comment (taehyub) [reply src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs]: 코멘트 감사합니다.

**Review by everLEEst — APPROVED** [review]
Looks good to me

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**